pwndbg/abi.py

```diff
@@ -111,3 +116,31 @@ class SigreturnABI(SyscallABI):
 linux_i386_srop = ABI(['eax'], 4, 0)
 linux_amd64_srop = ABI(['rax'], 4, 0)
 linux_arm_srop = ABI(['r7'], 4, 0)
+
+
+# See issue #264 for more reference
+# 'GNU/Linux' for linux
+# 'none' for bare metal
+abi = None
+
+def is_linux():
+    return 'Linux' in abi
+
+@pwndbg.events.start
+def update():
+    global abi
+
+    # detect current ABI of client side by 'show osabi'
+    osabi_string = gdb.execute('show osabi', to_string=True)
+    match = re.search('currently "([^"]+)"', osabi_string)
+    if match:
+        abi = match.group(1)
+
+def Linux(func):
```

Comment:
I'd rather rename this to `OnlyLinux` or `LinuxOnly` to be slightly more explanatory